src/rsz/src/Rebuffer.cc

```diff
@@ -227,55 +240,44 @@ RepairSetup::rebufferBottomUp(const BufferedNetPtr& bnet,
         junc->setCapacitance(p->cap() + q->cap());
         junc->setRequiredPath(min_req->requiredPath());
         junc->setRequiredDelay(min_req->requiredDelay());
-        Z.push_back(std::move(junc));
+        slacks[junc.get()] = slackPenalized(junc);
+        Z[base + j] = std::move(junc);
       }
     }
     // Prune the options if there exists another option with
     // larger required and smaller capacitance.
-    // This is fanout^2.
-    // Presort options to hit better options sooner.
+    // This is fanout*log(fanout) if options are
+    // presorted to hit better options sooner.
```

Comment:
I think both comments are wrong.  If the fanouts are evenly split in two between Z1 & Z2 then Z is size (fanout/2)^2.  The sort is big-O N*logN of N=fanout^2 in that case.

Reply:
The complexity is a function of two variables here, but I decided to keep the original intent. Hm, I think we can just avoid the issue saying that the problem is N*logN, without any explicit reference to fanout.